tox.ini

```diff
@@ -20,7 +20,9 @@ deps =
     test: pytest-cov
     test: pytest-timeout
 commands_pre =
+    # Build ioutil for both current python (e.g. 3.8) and platform python (e.g. 3.6)
     python setup.py build_ext --build-lib .
+    /usr/bin/python3 setup.py build_ext --build-lib .
```

Comment:
Will the 2 `setup.py build_ext --build-lib .` have different names?

Reply:
yes:

```
gcc -pthread -shared -Wl,-z,relro -Wl,-z,now -g -Wl,-z,relro -Wl,-z,now -g build/temp.linux-x86_64-3.8/ovirt_imageio/_internal/ioutil.o -L/usr/lib64 -o ./ovirt_imageio/_internal/ioutil.cpython-38-x86_64-linux-gnu.so
gcc -pthread -shared -Wl,-z,relro -Wl,-z,now -g -Wl,-z,relro -Wl,-z,now -g build/temp.linux-x86_64-3.6/ovirt_imageio/_internal/ioutil.o -L/usr/lib64 -lpython3.6m -o ./ovirt_imageio/_internal/ioutil.cpython-36m-x86_64-linux-gnu.so
```

Reply:
One more "corner case" :)
What if current Python version and platform Python versions are the same - won't there be a problem with `ioutil.cpython` override when running the newly added command?

Reply:
No, build_ext does not nothing in this case.

See the latest run on centos 8:
https://github.com/oVirt/ovirt-imageio/runs/7287479259?check_suite_focus=true